benchmarks/query_benchmarks/query_values/benchmark.py

```diff
@@ -0,0 +1,13 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryValues:
+    def setup(self):
+        bench_setup(migrate=True)
+
+    def teardown(self):
+        Book.objects.all().delete()
+
+    def time_query_values(self):
+        list(Book.objects.values("title"))
```

Comment:
Same here, let's repeat it 10x

https://github.com/smithdc1/django-asv/runs/6720503551?check_suite_focus=true#step:5:69